src/fmt.ml

```diff
@@ -65,6 +72,10 @@ let rec list ?(pp_sep = cut) pp_v ppf = function
 | v :: vs ->
     pp_v ppf v; if vs <> [] then (pp_sep ppf (); list ~pp_sep pp_v ppf vs)
 
+let hashtbl ?(pp_sep = cut) ~pp_k ~pp_v ppf tbl =
+  let f ppf k v = pf ppf "@[<1>%a:@.%a@]%a" pp_k k pp_v v pp_sep () in
```

Comment:
> Looking at this again I'm a little bit unconvinced by what we concluded the other day.

Well in fact I'm again convinced that it was the good conclusion... I did a rather big API overhaul in fd2e8d30d8d20d6354c44ced5245c and updated the online [api docs](http://erratique.ch/software/fmt/doc/Fmt).